### PR TITLE
[10.0][ADD] delivery_price_rule_untaxed

### PR DESCRIPTION
**Pull request by SimoRubi**
This module allows the user to use the total untaxed price of the sale order inside the rules used by delivery methods.

### CODE REVIEW

**Review by eLBati — APPROVED** [review]
code review

**Review by tafaRU — APPROVED** [review]
Only code review.

**Comment by SimoRubi** [comment]
@OCA/logistics-maintainers can this be merged?

File: delivery_price_rule_untaxed/models/delivery_carrier.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            if (test and 'untaxed_price' in
+                    (line.variable, line.variable_factor)):
+                price = (line.list_base_price
+                         + line.list_price * price_dict[line.variable_factor])
+                criteria_found = True
+                break
+            elif test:
+                return super(DeliveryCarrier, self) \
+                    .get_price_from_picking(total, weight, volume, quantity)
+        if not criteria_found:
```
Comment (pedrobaeza) [inline delivery_price_rule_untaxed/models/delivery_carrier.py]: I think you should call super outside the loop and you don't need this extra `raise`

**Review by rousseldenis — APPROVED** [review]
Code review. Seems good

**Comment by rousseldenis** [comment]
Oups, too fast. @SimoRubi Can you attend @pedrobaeza remarks ?

**Comment by SimoRubi** [comment]
@pedrobaeza and @rousseldenis thanks for your reviews!
I've followed your suggestions in https://github.com/OCA/delivery-carrier/pull/168/commits/3177d169b67291b79a7d9c41b6b4a6813197e60b

File: delivery_price_rule_untaxed/models/delivery_carrier.py (hunk 1 above)
Comment (SimoRubi) [reply delivery_price_rule_untaxed/models/delivery_carrier.py]: Done in https://github.com/OCA/delivery-carrier/pull/168/commits/3177d169b67291b79a7d9c41b6b4a6813197e60b

**Comment by SimoRubi** [comment]
Fixed everything up

**Comment by SimoRubi** [comment]
@OCA/logistics-maintainers is this ok to be merged?
Thanks

**Pull request merged**